app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt

```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
     task: Task
 ) {
 
+}
+
+@Composable
+fun SportTaskForm() {
+    var supportsDistanceMetrics by remember { mutableStateOf(false) }
+    var title by remember { mutableStateOf("") }
+    var duration by remember { mutableStateOf("") }
+    var startDate by remember { mutableStateOf(Date()) }
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White)
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            //horizontalAlignment = Alignment.CenterHorizontally
```

Comment:
If horizontal alignment is not required, then this redundant comment should be removed. Else, uncomment it to make it apply.